src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -864,7 +864,7 @@ static void Main()
         {
             throw new System.Exception();
         }
-        catch (System.Exception) when (false)
+        catch (System.Exception) when (default)
```

Comment:
Why did we delete the `false` test case here?  Sure `default` should compile to `false` in this case, but that's what testing should verify.  My initial reaction here was to wonder why we didn't have a test for `false` and `default`.